### DISCUSSION TITLE
"tabout" Plugin: Not Loading with ":Lazy" Command - Seeking Assistance

### DISCUSSION THREAD

**Discussion by zoonyoung** [NvChad/NvChad#2579] [Q&A]
I installed the ["tabout](https://github.com/abecodes/tabout.nvim)" plugin to use, but it's not working. When I execute the ":Lazy" command, it says "not loaded." I'm wondering how to resolve this issue. I'm planning to ask in an English community, so could you please help me phrase the question effectively?

**Comment by siduck** [comment]
@zoonyoung read the plugins docs in nvchad , it clearly mentions one thing..
